css/components/_input.css

```diff
@@ -0,0 +1,87 @@
+.input_group {
+  display: flex;
+  flex-direction: column;
+  padding: 0.5rem 1rem;
+}
+
+input{
+  border: 1px solid var(--grey-color-300);
+}
+
+.input_label {
+  margin-bottom: 0.5rem;
+}
+
+.input_mandatory {
+  border-color: var(--input-fail);
+  font-size: inherit;
+}
+
+.input_small {
+  font-size: 1rem;
+  padding: 0.25rem 0.5rem;
+}
+
+.input_regular {
+  font-size: 1.25rem;
+  padding: 0.5rem 0.625rem;
+}
+
+.input_large {
+  font-size: 1.5rem;
+  padding: 0.625rem 0.625rem;
```

Comment:
Just a small suggestion: if the value for top/bottom/left/right is same. use :
padding: 0.625rem;

It will give value to all.

Reply:
Thanks for pointing out second value should be 0.75 rem.
Fixed that.
Thanks man!!